public/deployments.json

```diff
@@ -88,7 +88,12 @@
         "endpoint": "classPath=com.dataintuitive.luciusapi.statistics"
       },
       "api": {
-        "url": "http://localhost:8090/jobs?context=luciusapi&appName=luciusapi&sync=true&timeout=30"
+        "url": "http://localhost:8090/jobs?context=luciusapi&appName=luciusapi&sync=true&timeout=30",
+        "asyncUrlStart": "http://localhost:8090/jobs?context=luciusapi&appName=luciusapi",
+        "asyncUrlStatus": "http://localhost:8090/jobs/",
+        "asyncStatusInterval": 0.5,
+        "longRunningTime": 5,
```

Comment:
Could you please make these defaults a bit more realistic? 10 and 30, respectively for instance?